pkg/operator/assets/bindata.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by go-bindata.
+// Code generated by go-bindata. DO NOT EDIT.
```

Comment:
Can you drop these changes to bindata.go; they are not required.

Reply:
yup, will do! Thanks